easy_profiler_core/profile_manager.h

```diff
@@ -223,50 +449,57 @@ class chunk_allocator
 
         if (!need_expand(n))
         {
-            int8_t* data = m_chunks.back().data + m_shift;
-            m_shift += n + sizeof(uint16_t);
+            // Temp to avoid extra load due to this* aliasing.
+            uint16_t chunkOffset = m_chunkOffset;
+            char* data = (char*)&m_chunks.back().data[0] + chunkOffset;
```

Comment:
What is the point of using `&data[0]` instead of just `data`?

Reply:
Good point. I probably put that there when I was just trying anything hoping to get it to work :) I'll change it.